openslides/motions/static/templates/motions/motion-detail/view-diff.html

```diff
@@ -11,21 +11,15 @@
                 <div class="action-row" ng-if="motion.isAllowed('can_manage')">
                     <div class="btn-group" data-toggle="buttons">
                         <label class="btn btn-sm btn-default" ng-class="{active: (change.status == 0)}"
-                               title="{{ 'Suggested' | translate }}" ng-click="change.status = 0; change.saveStatus();">
+                               title="{{ 'Accepted' | translate }}" ng-click="change.status = 0; change.saveStatus();">
```

Comment:
Here the same: I do not think that the default is accepted. The default has not specific decision and OpenSlides only provides to reject single recommendations explicitly. The rest is part of the motion and will be “accepted” if the motion in total is set to accepted.